corehq/apps/app_manager/feature_support.py

```diff
@@ -208,3 +208,7 @@ def ush_case_claim_2_53(self):
             toggles.USH_CASE_CLAIM_UPDATES.enabled(self.domain)
             and self._require_minimum_version('2.53')
         )
+
+    @property
+    def supports_ush_empty_case_list_text_2_54(self):
+        return self._require_minimum_version('2.54')
```

Comment:
Usually here is a nice place to also check for FF so it doesn't have to be a separate check, is that something you would consider?

Reply:
ha I see this change later on in the PR-- fantastic